src/lib/createPollBlocks.ts

```diff
@@ -31,6 +31,15 @@ export function createPollBlocks(block: BlockBuilder, question: string, options:
 
     block.addDividerBlock();
 
+    // Array of vote quantities
+    const voteQuantities = poll.votes.map(vote => vote.quantity)
```

Comment:
Why create this temp array - why not simply apply the reduce function here

Reply:
Had created for better code readability and to reuse variable for max within reduce. Removed in the update since we decided to store max quantity in a separate variable for use within reduce. Thanks.